src/main/scala/cromwell/binding/Declaration.scala

```diff
@@ -9,6 +9,7 @@ object Declaration {
     Declaration(
       scopeFqn,
       ast.getAttribute("type").wdlType(wdlSyntaxErrorFormatter),
+      Option(ast.getAttribute("postfix")) map {_.sourceString()},
```

Comment:
spaces inside braces. also since sourceString isn't side effect you shouldn't use parens